terraform/modules/ec2-task-definition/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "name_prefix" {
+  type        = string
+  description = "(Required) Name prefix for resources we create, defaults to repository name"
+}
+variable "region" {
+  type        = string
+  description = "(Required) The region to deploy into"
+}
+variable "chrome_image" {
+  type = string
+}
+variable "guacd_image" {
+  type = string
+}
+variable "jupyterhub_image" {
+  type = string
+
```

Comment:
this empty line shouldn't be here, have you run `terraform fmt -recursive` from the top directory?